docs/node/mainnet/eden-upgrade.md

```diff
@@ -381,7 +394,7 @@ oasis-node genesis migrate --genesis.new_chain_id oasis-4
 #### Random Beacon
 
 * **`beacon.base`** is the network's starting epoch. It will be set to the epoch
-  of Mainnet's state dump + 1, `TBD`.
+  of Mainnet's state dump + 1, `28018`.
```

Comment:
```suggestion
  of Mainnet's state dump + 1, `28017`.
```

Reply:
why does it then say of Mainnet's state dump + 1?

Reply:
fixed